app/controllers/reviews_controller.rb

```diff
@@ -34,7 +34,7 @@ def update
   def destroy
     @neighborhood = Neighborhood.find(params[:neighborhood_id])
     @review = Review.find(params[:id])
-    if @review.user == current_user
+    if @review.user == current_user || current_user.admin?
```

Comment:
consider instead `@review.destroyable_by?(current_user)` in the same way I indicated above.